beetmoverscript/templates/maven_geckoview.yml

```diff
@@ -0,0 +1,62 @@
+---
+metadata:
+    name: "Maven repository"
+    description: "Maps artifacts to spec'd maven location"
+    owner: "release@mozilla.com"
+
+s3_bucket_path: maven2/org/mozilla/{{ artifact_id }}/{{ version }}/  # Maven groupId is org.mozilla
+
+mapping:
+{% for locale in ['en-US'] %}
+  "{{ locale }}":   # Locale is not needed for geckoview, it's used by move_beets, though
```

Comment:
Yeah, I feel you. However this is going to be washed away when declarative artifacts is done.